--- pyxf86config-0.3.37/Makefile.am	2008-03-11 19:47:42.000000000 +0100
+++ pyxf86config-0.3.37.new/Makefile.am	2008-08-30 02:15:10.505157116 +0200
@@ -2,11 +2,11 @@
 
 pythondir = $(libdir)/python@PYTHON_VERSION@/site-packages
 
-python_PROGRAMS = ixf86configmodule.so
+python_LTLIBRARIES = ixf86configmodule.la
 python_DATA = xf86config.py
 
 EXTRA_DIST=xf86config.py
 
-ixf86configmodule_so_SOURCES = pyxf86conf.h pyxf86conf.c xf86ParserExt.h xf86config_ext.c
-ixf86configmodule_so_LDFLAGS = -module -shared @GOBJECT_LIBS@
-ixf86configmodule_so_LDADD = @X_LIBS@ -lxf86config
+ixf86configmodule_la_SOURCES = pyxf86conf.h pyxf86conf.c xf86ParserExt.h xf86config_ext.c
+ixf86configmodule_la_LDFLAGS = -avoid-version -module -shared @GOBJECT_LIBS@
+ixf86configmodule_la_LIBADD = @X_LIBS@ -lxf86config
